spring-kafka-test/src/main/java/org/springframework/kafka/test/assertj/KafkaConditions.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 the original author or authors.
+ * Copyright 2017 the original author or authors.
```

Comment:
Exactly! Like you did here: https://github.com/spring-projects/spring-kafka/pull/241/files#diff-be360e97e458e9f0b6a7aea7890dbb8eR2